crates/core/src/vm.rs

```diff
@@ -441,11 +429,11 @@ impl ProgramVm for DbProgram<'_, '_> {
     }
 
     fn env(&self) -> &EnvDb {
-        &self.env
+        unreachable!()
     }
 
     fn env_mut(&mut self) -> &mut EnvDb {
-        &mut self.env
+        unreachable!()
     }
```

Comment:
Let's remove these from the trait entirely right now, since `DbProgram` no longer has an `EnvDb`.